### DISCUSSION TITLE
How do I identify a test that has re-run

### DISCUSSION THREAD

**Discussion by ChristopherDrum** [lyndsey-ferguson/fastlane-plugin-test_center#352] [Q&A]
Trying out multi-scan today, and I'm not sure what I'm looking at in the logs.
I see at the end of the logs (in CircleCI) `total_retry_count | 12 `
However, looking through the report, I don't see any indication of which specific tests were retried.
This number `12` is higher than I expected, so it is important to me to understand what is happening.
Are there any specific phrases or keywords I should look for? Searching for `retry` only turns up the one line above.

**Comment by lyndsey-ferguson** [comment]
The retry count of each test should be an attribute for a test case in the Junit XML file that was exported. Let me know if you don't see it there.

**Reply by lyndsey-ferguson** [reply]
Did this help you @ChristopherDrum ?
